tests/tests-ui/cypress/support/login/commands_login.js

```diff
@@ -0,0 +1,30 @@
+Cypress.Commands.add("loginUI", (user, password) => { 
+    if(user == null){
+		user = Cypress.env('user');
+		password = Cypress.env('password');
+	}
+	login(user,password);
+	cy.get('.header-menu-buttons').should('contain', user)
+})
+
+Cypress.Commands.add("loginError", (user,password) => {
+	login(user,password);
+	cy.get('.callout').should('contain', 'Incorrect username or password');
+})
+
```

Comment:
If I understood this command correctly, it tests if the message contains that specific text. This test will fail if we translate the message (and we are). Can you check for status code instead maybe?

Reply:
when the login will merge ill check it, for now when we dont yet have login its not called